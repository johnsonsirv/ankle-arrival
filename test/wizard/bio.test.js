/* eslint-disable react/jsx-props-no-spreading */
/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import BioPage from '../../src/wizard/bio';

function setup() {
  const props = {
    onInputChange: jest.fn(),
    guestName: 'Joe Smith',
  };

  const enzymeWrapper = shallow(<BioPage {...props} />);
  return { props, enzymeWrapper };
}

describe('<BioPage /> rendering', () => {
  const { enzymeWrapper } = setup();
  it('renders correctly', () => {
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
