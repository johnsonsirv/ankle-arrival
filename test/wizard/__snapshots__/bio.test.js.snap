// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BioPage /> rendering renders correctly 1`] = `ShallowWrapper {}`;
